qiskit_nature/problems/second_quantization/lattice/lattices/lattice.py

```diff
@@ -25,11 +25,11 @@
 from qiskit.utils import optionals as _optionals
 
 if _optionals.HAS_NETWORKX:
-    # pylint: disable=unused-import
+    # pylint: disable=import-error,unused-import
```

Comment:
Why do we need to add this pylint exception?

Reply:
Matplotlib is an optional library. If you don’t have matplotlib installed and run pylint in your system, it will fail with import-error. This cover that case. It doesn’t happen in the CI because pylint runs there with all optional libraries installed.